spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+RSpec.describe "Foods page" do
+  
+  it "shows all foods that match a search" do
+    visit root_path
+    fill_in :q, with: "Sweet Potatoes"
+    click_button 'Search'
+    expect(current_path).to eq(foods_path)
+    expect(page).to have_content('Results:')
+    expect(page).to have_content(44128)
+    expect(page).to have_content('SWEET POTATO')
```

Comment:
Definitely needed a bit more in terms of testing all  the attributes and the number of foods (10) the spec was asking for.